libs/server/core/application-services/src/lib/email/email.service.ts

```diff
@@ -37,9 +37,14 @@ export class EmailService {
   }
 
   sendEmailVerification(email: string, hash: string, registerAs?: SignUpDto['registerAs']) {
-    if (this.shouldNotSendNotification) return;
-
-    const url = `${environment.appUrl}${this.route.path.verifyEmail.ROOT}?email=${email}&hash=${hash}&register=${registerAs}`;
+    const url = `${environment.appUrl}${this.route.path.verifyEmail.ROOT}` +
+      `?email=${encodeURIComponent(email)}` +
+      `&hash=${hash}` +
+      `&register=${registerAs}`;
+    if (this.shouldNotSendNotification) {

```

Comment:
this enables printing of url to console in local dev for testing
(since local dev does not have an email client)